modules/apps/site-navigation/site-navigation-menu-web/src/main/resources/META-INF/resources/configuration.jsp

```diff
@@ -235,10 +235,12 @@ else {
 			<clay:col
 				md="6"
 			>
-				<liferay-portlet:preview
-					portletName="<%= portletResource %>"
-					showBorders="<%= true %>"
-				/>
+				<nav class="navbar navbar-classic navbar-expand-md navbar-light pb-3">
```

Comment:
These classes are added outside the Navigation Menu. I don't think the preview should be responsible for those styles.

If we want the preview to match the main nav in classic, we should add an ADT and use that instead of https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-classic/src/templates/portal_normal.ftl#L55-L67

Reply:
A feature that would be nice is being able to pass `cssClass` to the parent element through the configuration menu. This would eliminate the need for a new ADT or custom wrapper element any time we want to change the color.

Reply:
> These classes are added outside the Navigation Menu. I don't think the preview should be responsible for those styles.

I would agree with @pat270... but reading through his own snippet https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-classic/src/templates/portal_normal.ftl#L55-L67 made me rethink it 😂 

The `navigation` widget already provides ADTs (actually templates) for the different view types. The fact that we're wrapping the widget in `classic` means that's what used to be our API and removing it would be a breaking change and would force us to update all navigation templates. 

Being that the case, the fix makes sense to me, since the expectation is that the navigation is wrapped in those classes. 

We can try to improve that in the next release, following both @eduardoallegrini and @pat270 comments.

Reply:
@jbalsas Either way, we're breaking the preview for other navs. If someone used this widget in the footer for their social nav, the preview will be wrapped in `<nav class="navbar navbar-classic navbar-expand-md navbar-light pb-3">`.

In styled theme, `navbar-classic` doesn't exist. It doesn't make sense to hardcode a fix for classic in a widget that is mostly theme agnostic.

Reply:
You're 100% right, @pat270! I missed that bit about the `classic` class in there... suggestions for a proper fix, then... that doesn't involve changing all the navigations? 😂

Reply:
I think the best solution is to take the markup in classic and add an ADT in https://github.com/liferay/liferay-portal/tree/master/modules/apps/site-navigation/site-navigation-menu-web/src/main/resources/com/liferay/site/navigation/menu/web/portlet/template/dependencies/. Maybe we can name it `portlet_display_template_navbar_classic.ftl`?

In `portal_normal.ftl` we can do:
```
<#assign preferences = freeMarkerPortletPreferences.getPreferences({
  "portletSetupPortletDecoratorId": "barebone",
  "displayStyle": "ddmTemplate_NAVBAR_CLASSIC-FTL"
}) />
<@liferay.navigation_menu default_preferences="${preferences}" />
```